roles/telegraf/files/run-telegraf.sh

```diff
@@ -237,27 +251,27 @@ function main {
   fi
 
   assert_is_installed "consul"
-  assert_is_installed "consul-template"
 
   wait_for_consul
 
-  local readonly enabled=$(consul_kv_with_default "${consul_prefix}telegraf/${type}/enabled" "no")
+  local readonly enabled=$(consul_kv_with_default "${consul_prefix}telegraf/server_types/${type}/enabled" "no")
 
   if [[ "$enabled" != "yes" ]]; then
     log_info "Telegraf is not enabled for ${type}"
   else
-    if [[ "$skip_template" == "false" && -f "$conf_template" ]]; then
-      log_info "Applying consul-template on \"$conf_template\" to generate \"$conf_out\"..."
-      consul-template -template "$conf_template:$conf_out" -once
-      log_info "consul-template applied successfully!"
-    fi
-
     enable_telegraf "$type" "$service_override_dir"
-    add_statsd_conf "$type" "$consul_conf" "$nomad_conf" "$vault_conf"
 
-    # consul must be restarted since we need it to be up first before we can use consul-template
-    # and now we are adding in new configuration
-    restart_service "consul"
+    local readonly elasticsearch=$(consul_kv_with_default "${consul_prefix}telegraf/server_types/${type}/output/elasticsearch/enabled" "no")
+
+    if [[ "$elasticsearch" == "yes" ]]; then
+      assert_is_installed "consul-template"
+      local readonly elasticsearch_service=$(consul_kv "${consul_prefix}telegraf/server_types/${type}/output/elasticsearch/service_name")
+
+      log_info "Configuring Telegraph to output to Elasticsearch at service name '${elasticsearch_service}'"
```

Comment:
`Telegraph` should be `Telegraf` haha

Reply:
Fixed in #109. You should look at #109 first.